iib/workers/tasks/build_merge_index_image.py

```diff
@@ -397,6 +397,8 @@ def is_bundle_version_valid(bundle_path, valid_ocp_version):
         raise IIBError(f'Invalid OCP version, "{valid_ocp_version}", specified in Index Image')
     try:
         bundle_version_label = get_image_label(bundle_path, 'com.redhat.openshift.versions')
+        if bundle_version_label is None:
+            raise ValueError
```

Comment:
I believe it would be better to raise an `IIBError` since it will be caught by API to properly log, also please consider including an error message for the error.

Reply:
Please check again. I made the change according to that.

Reply:
Great, thanks!